app/views/users/show.html.erb

```diff
@@ -1 +1,24 @@
-<h1>Show user</h1>
\ No newline at end of file
+<section class="users-show">
+    <div>
+        <img src="<%= @user.photo %>" />
+        <h1><%= @user.name %></h1>
+        <p>Number of posts:<%= @user.posts_counter || 0 %></p>
+        <p><%= @user.bio %></p>
+    </div>
+
+ 
+</section>
+   <div>
+    <% @user.posts.each do |e| %>
```

Comment:
- [ ] Kindly use the `recent_posts` method instead of displaying the entire user posts in the `users#show` view as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Use%20methods%20that%20you%20have%20created%20in%20the%20Blog%20App%20%2D%20processing%20data%20in%20models.). They asked to use the methods that already created in the `Blog App - processing data in models` project.